src/js/components/Form/Form.js

```diff
@@ -13,12 +13,39 @@ const defaultMessages = {
 };
 const defaultValue = {};
 const defaultErrors = {};
+const defaultInfos = {};
+
+const updateErrors = (nextErrors, name, error) => {
+  // we disable no-param-reassing so we can use this as a utility function
+  // to update nextErrors, to avoid code duplication
+  /* eslint-disable no-param-reassign */
+  if (
+    (typeof error === 'object' && error.status === 'error') ||
+    typeof error === 'string'
+  ) {
+    nextErrors[name] = typeof error === 'object' ? error.message : error;
+  } else {
+    delete nextErrors[name];
+  }
+  /* eslint-enable no-param-reassign */
+};
+
+const updateInfos = (nextInfos, name, error) => {
+  /* eslint-disable no-param-reassign */
+  if (typeof error === 'object' && error.status === 'info') {
+    nextInfos[name] = error.message;
+  } else {
+    delete nextInfos[name];
+  }
+  /* eslint-enable no-param-reassign */
+};
 
 const Form = forwardRef(
   (
     {
       children,
       errors: errorsProp = defaultErrors,
+      infos: infosProp = defaultInfos,
```

Comment:
on doc.js this prop is called `info`, and not `infos`.

Reply:
Actually, it's `info` for FormField and `infos` for Form. I've added the missing `infos` to Form/doc.js.